week-2-wide-perspective/w02e05-simple-server-side/src/validators/order.ts

```diff
@@ -0,0 +1,15 @@
+import { createValidator } from "express-joi-validation";
```

Comment:
Przemyślałeś również temat walidowania `body` i requestów.
Fajnie.

To co mnie w dłuższej perspektywie "męczyło" to podział kubełkowy "services" oraz "validators".
Zauważ, że jeśli byśmy rozwijali projekt - to musielibyśmy biegać pomiędzy services, a validators a np. `v1` i "controllers"

Fajniejszy byłby podział "feature like", sprawdź:
https://alexkondov.com/tao-of-node/